content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/starshipit_shipping.rst

```diff
@@ -0,0 +1,201 @@
+automatically get rates from specific couriers (such as Australia Post, NZ Post, DHL,...)
+based on predefined conditions.
+
+Setup in Starshipit
+===================
+
+Create an account and activate carriers
+---------------------------------------
+
+To get started, go to `Starshipit's platform <https://starshipit.com/>`_ to configure the account
+and generate the connector credentials. Log in with the Starshipit account, or create a new one if
+needed.
+
+Pickup address configuration
+----------------------------
+
+Once logged into the Starshipit account, navigate to :menuselection:`Settings --> Pickup address`,
+and fill in the field for :guilabel:`Pickup address`. It should be the same address as the
+warehouse you are shipping from.
+
+.. image:: starshipit_shipping/starshipit-settings-address.png
+   :align: center
+   :alt: Adding addresses in the Starshipit settings.
+
+Couriers configuration
+----------------------
+
+Next, navigate to :menuselection:`Settings --> Couriers`,
+and add the :guilabel:`Couriers` you wish to integrate with.
+
+.. image:: starshipit_shipping/starshipit-settings-couriers.png
+   :align: center
+   :alt: Adding addresses in the Starshipit settings.
+
+.. tip::
+   You can find details on how to integrate with the different couriers in
+   `Starshipit's support center <https://support.starshipit.com/hc/en-us/>`_.
+
+Couriers services configuration
+-------------------------------
+
+In the checkout rates settings, you will need to set which courier services you want to provide in
+Odoo.
+
+.. image:: starshipit_shipping/starshipit-checkout-rate.png
+   :align: center
+   :alt: Checkout rates in the Starshipit settings.
+
+Shipping rules configuration
+----------------------------
+
+Finally, you will need to configure shipping rules in Starshipit. These are used when placing a
+shipping order to set the correct courier on your order. There are multiple ways to do so. We
+recommend adding a rule with the condition being ":guilabel:`contains`", the value being
+":guilabel:`the product code`", and the action "setting that product code and courier" to the
+shipping order.
+
+.. image:: starshipit_shipping/starshipit-rules.png
+   :align: center
+   :alt: Shipping rules in the Starshipit settings.
+
+Finding Starshipit API credentials
+----------------------------------
+
+In your Starshipit account, navigate to :menuselection:`Settings --> API` in the side menu.
+There, you can find the API keys you will need to connect to Odoo.
+
+.. image:: starshipit_shipping/starshipit-settings-api.png
+   :align: center
+   :alt: Finding the Starshipit API keys.
+
+Setup in Odoo
+=============
+
+Install the Starshipit shipping module
+--------------------------------------
+
+After the Starshipit account is set up and configured, it's time to configure the Odoo database.
+To get started, go to Odoo's :guilabel:`Apps` module,  search for the :guilabel:`Starshipit
+Shipping` integration, and install it.
```

Comment:
I couldn't find Starshipit module in the 17.0 or master runbot.

![image](https://github.com/odoo/documentation/assets/23611711/a5a4aad8-aa95-4ef6-b3b2-5acc37bdb84a)

Am I searching for it properly?

Reply:
Hello! The module is on a pull request linked to this one so that both get released at the same time.
https://58697424-17-0-all.runbot114.odoo.com/web#id=21&cids=6&menu_id=1170&action=1482&model=delivery.carrier&view_type=form

Here you can find a runbot with the app installed.
To ease your test, I went and created a starshipit shipping method with our test API keys.
The AU Company, and the partner Deco Addict have been set up with addresses that are correct to be used for testing.
I recommend keeping sendle drop off as shipping service as it's easier for me to cancel orders afterward :)

Just let me know when you are done.
Thanks!

Reply:
Also if you need access to the starshipit website to check there too, let me know and I'll share the credentials with you on discord :smile:

Reply:
Thank you so much for getting back to me so quickly!

Reply:
@Felicious Thank you for the review! I will be working with our Australian colleagues during the day to address your comments and most likely ask for another review afterward :pray: